content/go/concepts/math-functions/terms/exp2/exp2.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: 'Exp2()'
+Description: 'Returns the value of base-2 exponential of given number.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Exp2()`** function returns the value of base-2 exponential of given number of type `float64`.
+
+## Syntax
+
+```pseudo
+import math
+
+result := math.Exp2(x)
+```
+
+Where `result` will contain the value of base-2 exponential of `x`, returned as type `float64`.
+Special cases:
+
+- The result of `Exp2(+Inf)` is +Inf
+- The result of `Exp2(NaN)` is NaN
+  
+## Example
+
```

Comment:
There should be a brief introductory sentence before the example

Reply:
For example?

Reply:
This is typical in our entries. Please review other docs if you need a reference.

Reply:
I am doing it with reference to other similar docs